### PR TITLE
Update contribution_tools.md

### PR DESCRIPTION
**Pull request by MamarezaAlipour**
VS Code is a Code editor, not an IDE!

### CODE REVIEW

**Review by jonmeow — CHANGES_REQUESTED** [review]
Can you please explain the distinction? VS Code supports code editing, building, and debugging: what's missing that you don't consider it an IDE?

**Comment by MamarezaAlipour** [comment]
> Can you please explain the distinction? VS Code supports code editing, building, and debugging: what's missing that you don't consider it an IDE?

As you know, VS Code doesn't do anything by itself. All these tasks that you mentioned are done by plugins. Basically, we call an IDE an environment that does these things in addition to several other services, specially and by default for a language / framework (single tool belt), and all those things are published exclusively by the IDE publishing company (expert support is guaranteed).

**Review by jonmeow — APPROVED** [review]
I'm not sure I think there's the same distinction, but per https://code.visualstudio.com/docs/supporting/faq#_what-is-the-difference-between-visual-studio-code-and-visual-studio-ide it seems VS Code is trying to brand itself a code editor so approving.

File: docs/project/contribution_tools.md [hunk 1]
```diff
@@ -322,7 +322,7 @@ Our recommended way of installing is to use
 
 ### Visual Studio Code
 
-[Visual Studio Code](https://code.visualstudio.com/) is an IDE used by several
+[Visual Studio Code](https://code.visualstudio.com/) is a code editor used by several
 of us. We provide [recommended extensions](/.vscode/extensions.json) to assist
 Carbon development. Some settings changes must be made separately:
```
Comment (jonmeow) [inline docs/project/contribution_tools.md]:
```suggestion
[Visual Studio Code](https://code.visualstudio.com/) is a code editor used by
several of us. We provide [recommended extensions](/.vscode/extensions.json) to
assist Carbon development. Some settings changes must be made separately:
```

Note the pre-commit failure, this is just formatting.

**Review by jonmeow — APPROVED** [review]

**Pull request merged**